.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
I think you use the develop branch for PRs and master for releases.
In that case it might make sense to run the build workflow in develop, or both?